.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module",
+    "project": ["tsconfig.json"]
+  },
+  "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "import/extensions": [
+      "error",
+      "always",
+      {
+          "pattern": {
+              "js": "never",
+              "jsx": "never",
+              "tsx": "never"
+          }
+        }
+   ],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "linebreak-style": 0
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- [ ] Please, Remove this file and use `.eslintrc.sjon` provided by Microverse [.eslintrc.json](https://github.com/Bondok6/world-data/blob/main/.eslintrc.json)

> I noticed it's not the same, Linter is working fine on my end when I removed this part. 👍
```
    "import/extensions": [
      "error",
      "always",
      {
          "pattern": {
              "js": "never",
              "jsx": "never",
              "tsx": "never"
          }
        }
   ],
```
> So please use this  [.eslintrc.json](https://github.com/Bondok6/world-data/blob/main/.eslintrc.json)